kolibri/plugins/example_plugin/assets/example/example_plugin.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
-var Kolibri = require('kolibri.core.KolibriCoreFrontEnd');
+var _ = require('lodash');
+var logging = require('loglevel');
 
-module.exports = {};
+var Plugin = require('plugin_base');
+
+var ExamplePlugin = Plugin.extend({
+
+    events: {
+        'something_happened': 'hello_world'
+    },
+    hello_world: function(plugin) {
+        if (plugin === this) {
```

Comment:
Why is this `if` statement required? So this callback is not guaranteed to be called on this plugin's context?